chalice/app.py

```diff
@@ -167,6 +215,10 @@ def __init__(self, view_function, view_name, path, methods,
         #: e.g, '/foo/{bar}/{baz}/qux -> ['bar', 'baz']
         self.view_args = self._parse_view_args()
         self.content_types = content_types
+        if cors is True:
```

Comment:
Can you add a comment about why the identity check is being used instead of just boolean comparison.  This is probably gonna catch other dev's attention because it's not a common pattern so it's worth explaining why we need to do this.